public/app/features/dashboard/dashgrid/PanelStateWrapper.tsx

```diff
@@ -624,6 +628,10 @@ export class PanelStateWrapper extends PureComponent<Props, State> {
     const { errorMessage, data } = this.state;
     const { transparent } = panel;
 
+    if (panelLogger.isDebugEnabled()) {
```

Comment:
Sorry why this IF? Isn't this handled by appenders?

Reply:
might be nice for the first arg to be a string or function.  then you don't need an if statement for doing costly string building

Reply:
@scottlepp yea, was thinking of that as well. perf cost of creating a function should be low enough for that to be preferable to this that requires 3 lines of code